src/mixins/KonvaRegion.js

```diff
@@ -35,7 +34,7 @@ export const KonvaRegionMixin = types.model({})
     };
   })
   .actions(self => {
-    let deferredSelectId = null;
+    let deferredSelectId;
```

Comment:
are there some importance in this change from `null` to `undefined`?

Reply:
Nope, it's just an accident